brightray/common/application_info_win.cc

```diff
@@ -62,4 +64,22 @@ bool GetAppUserModelID(ScopedHString* app_id) {
   return app_id->success();
 }
 
+bool IsRunningInDesktopBridgeImpl() {
+  // GetPackageFamilyName is not available on Windows 7
+  if (IsWindows8OrGreater()) {
```

Comment:
C++ is not JavaScript, (un)fortunately. This will still make Electron.exe statically linked to GetPackageFamilyName and show an error dialog when running on Windows 7. You need to use GetProcAddress. cc @alespergl